Datasnap/Classes/Utilities/Datasnap.m

```diff
@@ -172,21 +185,20 @@ - (void)onDataInitialized
 - (void)trackEvent:(BaseEvent*)event
 {
     event.dataSnapVersion = [[[NSBundle mainBundle] infoDictionary] objectForKey:@"CFBundleShortVersionString"];
-    NSDateFormatter* dateFormatter = [[NSDateFormatter alloc] init];
+    NSDateFormatter* dateFormatter = [NSDateFormatter new];
     [dateFormatter setDateFormat:@"yyyy-MM-dd'T'HH:mm:ssZZZZZ"];
     event.created = [NSString stringWithFormat:@"%@", [dateFormatter stringFromDate:[NSDate date]]];
```

Comment:
No need to use stringWithFormat here.  Would also put this into the custom init for events